controllers/hetznercluster_controller_test.go

```diff
@@ -902,17 +910,19 @@ func TestSetControlPlaneEndpoint(t *testing.T) {
 				},
 			},
 		}
+
 		got := setControlPlaneEndpoint(hetznerCluster)
-		// ControlPlaneEnpdoint should still remain nil
-		if (hetznerCluster.Spec.ControlPlaneEndpoint != nil){
+
+		if hetznerCluster.Spec.ControlPlaneEndpoint != nil{
```

Comment:
this looks weird.. there should always be a space between nil and {... Is it possible that you don't have linting and formatting in place that fixes stuff like that? Please have a look into that. You can e.g. configure vscode in a way that it formats your code on save.

Reply:
You are right. I do use it. However, I thought the code was already formatted for, hence did not do it. I'll make sure I use it before committing next time